templates/_layout.html

```diff
@@ -44,6 +44,7 @@
 
     {% include "_footer.html" %}
 
+    <script src="{{ static_url('js/dist/base.js') }}"></script>
```

Comment:
@Lukewh I added the `base.js` script to layout.
But, as sentry is included in the template, I guess we would need to include this script after that, right?
So that will mean we need to move setting up raven to layout. To make sure raven is loaded even if page itself doesn't have any custom scripts?

Reply:
Yeah we could do something like:
`_layout.html`:
```
<script src="base"></script>
{% block scripts %}{% endblock %}
<script src="raven"></script>
<script>
 Raven.setupStuff(config, function() {
   {% block script_execs %}{% endblock %}
 });
</script>
```

`market.html` etc.:
```
{% block scripts %}
<script src="d3"></script>
<script src="publisher.js"></script>
{% endblock %}
{% block script_execs %}
  snapcraft.publisher.market(...)
{% endblock %}
```

Not great, but means we can wrap what we need to with the raven stuff?